README.md

```diff
@@ -9,3 +9,12 @@ To report a problem or browse the list of open bugs, please visit the
 For more information, including instructions for compiling, installing, and
 hacking on DMD, check the [contribution guide](CONTRIBUTING.md) and
 visit the [D Wiki](http://wiki.dlang.org/DMD).
+
+All significant contributors to DMD source code, via github, bugzilla, email,
+wiki, the D forums, etc., please assign copyright to those
+DMD source code changes to the D Language Foundation. Please send
+an email to walter@digitalmars.com with the statement:
+
+"I hereby assign copyright in my contributions to DMD to the D Language Foundation"
+
+and include your name and date.
```

Comment:
Yes, this will help reviewers, as they can just check that list and ask a new contributor to send that email first.